src/benchq/data_structures/algorithm_description.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+from .quantum_program import QuantumProgram
+
+
+@dataclass
+class AlgorithmDescription:
+    program: QuantumProgram
+    n_calls: int
```

Comment:
What is n_calls here? I don't think I've seen it used?

Reply:
It's not being used, but added it to set it up for future use.